instrumentation/inslogger/inslogger_ext_test.go

```diff
@@ -12,14 +12,15 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-///
+//
```

Comment:
these stylistic changes in copyright notice annoying, increase time of review.

Reply:
@ita-sammann whats is the right format in this case pls?

Reply:
The correct format is 2 slashes

Reply:
ok